lib/vector-buffers/src/variants/disk_v2/tests/size_limits.rs

```diff
@@ -161,8 +161,7 @@ async fn writer_waits_when_buffer_is_full() {
                 .was_closed()
                 .finalize();
 
-            let mut second_record_read =
-                spawn(async { reader.next().await.expect("read should not fail") });
+            let mut second_record_read = spawn(async { read_next(&mut reader).await });
```

Comment:
Same comment about collapsing this to avoid the async block and using `read_next` directly.

Reply:
Ah, that one could probably do, yes.